.commitlintrc.json

```diff
@@ -7,6 +7,7 @@
       2,
       "always",
       [
+	"k8sutil",
```

Comment:
The prefixes should be alphabetized in this list for maintainability

Reply:
@travisn  wrote:
> The prefixes should be alphabetized in this list for maintainability

done